packages/nimble-components/src/anchor-tabs/tests/anchor-tabs.stories.ts

```diff
@@ -0,0 +1,132 @@
+import { html, ref, repeat, when } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { withXD } from 'storybook-addon-xd-designs';
+import { createUserSelectedThemeStory } from '../../utilities/tests/storybook';
+import '../../all-components';
+import type { AnchorTabs } from '..';
+
+interface TabsArgs {
+    tabs: TabArgs[];
+    activeId: string;
+    toolbar: string;
+    tabsRef: AnchorTabs;
+    onTabClick: (tabs: AnchorTabs, tabId: string) => boolean;
+    onTabKeypress: (tabs: AnchorTabs, tabId: string, event: Event) => boolean;
+}
+
+interface TabArgs {
+    label: string;
+    id: string;
+    href: string;
+    disabled: boolean;
+}
+
+const overviewText = `Anchor tabs are a sequence of links that are styled to look like tab elements, where one link can
+be distinguished as the currently active item. Use this component instead of the standard tabs component when each tab
+represents a different URL to navigate to. Use the standard tabs component when the tabs should switch between different
+tab panels hosted on the same page.`;
+
+const metadata: Meta<TabsArgs> = {
+    title: 'Anchor Tabs',
+    decorators: [withXD],
+    parameters: {
+        docs: {
+            description: {
+                component: overviewText
+            }
+        },
+        design: {
+            artboardUrl:
+                'https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/b2aa2c0c-03b7-4571-8e0d-de88baf0814b/specs'
+        }
+    },
+    // prettier-ignore
+    render: createUserSelectedThemeStory(html`
+        <nimble-anchor-tabs ${ref('tabsRef')} activeid="icons">
+            ${when(x => x.toolbar, html<TabsArgs>`<nimble-tabs-toolbar :innerHTML="${x => x.toolbar}"></nimble-tabs-toolbar>`)}
+            ${repeat(x => x.tabs, html<TabArgs, TabsArgs>`
+                <nimble-anchor-tab
+                    id="${x => x.id}"
+                    ?disabled="${x => x.disabled}"
+                    href="${x => x.href}"
+                    target="iframe"
+                    @click="${(x, c) => c.parent.onTabClick(c.parent.tabsRef, x.id)}"
+                    @keypress="${(x, c) => c.parent.onTabKeypress(c.parent.tabsRef, x.id, c.event)}"
+                >${x => x.label}</nimble-anchor-tab>
+            `)}
+        </nimble-anchor-tabs>
+        <iframe name="iframe" src="${x => x.tabs[0]!.href}"></iframe>
```

Comment:
Talked in person and agreed to remove the iframe part of the example and make the tabs point to external links. Also make the activeid configurable with a select / dropdown but set the default to none.

Reply:
Updated the stories for anchor tabs and standard tabs (to be consistent) as we had discussed. I added an `activeId` radio to the standard tabs as well, because it is part of the public API and prevents always reseting activeId to the first tab when another setting is changed, e.g. disabling the first tab. However this has the downside of getting out-of-sync with the visuals if you change the active tab by user interaction e.g. clicking. I think that's still better than leaving it out.